packages/js/plugins/ethereum/web3api.plugin.yaml

```diff
@@ -1,3 +1,6 @@
 format: 0.0.1-prealpha.1
 language: plugin/typescript
 schema: ./schema.graphql
+import_redirects:
+  - uri: "ens/logger.core.web3api.eth"
```

Comment:
We dont have the logger interface deployed?

Reply:
Yes we do but we use redirects instead for faster testing,
Also this is just debug leftover - nice catch.